python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -169,7 +169,7 @@ def _executeJob(
         quantum = pickle.loads(quantum_pickle)
         report: QuantumReport | None = None
         try:
-            _, report = quantumExecutor.execute(task_node, quantum)
+            quantum, report = quantumExecutor.execute(task_node, quantum)
```

Comment:
@TallJimbo  Is there a reason this was `_` before? I can put it back (I was experimenting with adding a cast around that `pickle.loads` and forgot to back this line out.

Reply:
I think it wasn't used until the `QuantumReport` stuff below was added, which was after I created this branch.